google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -232,6 +232,8 @@ int main(int argc, char* argv[]) {
     std::lock_guard<std::mutex> lk(mu);
     gcs_bm::PrintAsCsv(std::cout, result);
     if (!result.status.ok()) {
+      std::cout << std::flush;
+      std::cerr << std::flush;
```

Comment:
> The benchmark ... was not flushing stdout before printing these logs, making it hard to interpret the output.

Might not this be true of any program calling `google::cloud::LogSink::Instance().Flush()`, and therefore that a remedy would be better placed within its implementation?  That is, `google::cloud::internal::StdClogBackend` is where we're actually dealing with `std::clog` and flushing.

Reply:
Good idea, done.